templates/sync-function/validation-module.js

```diff
@@ -27,445 +27,452 @@ function validationModule(utils, simpleTypeFilter, typeIdValidator) {
     return nameComponents.join('');
   }
 
-  // Resolves a constraint defined at the document level (e.g. `propertyValidators`, `allowUnknownProperties`, `immutable`).
-  function resolveDocConstraint(doc, oldDoc, constraintDefinition) {
```

Comment:
This file may best be viewed with `?w=1` since the bulk of changes are from scope (and thus indentation) adjustments.